README.md

```diff
@@ -30,6 +30,12 @@ The **advanced search** finder provides facetted searching of GOV.UK taxonomy an
 
 [Read more about the advanced search finder](docs/advanced-search-finder.md)
 
+The EU exit business readiness finder ('business finder') provides a links-based facetted search of GOV.UK Brexit-related content for businesses:
+
+* [gov.uk/find-eu-exit-guidance-business](https://www.gov.uk/find-eu-exit-guidance-business)
+
+Read [more about the business finder](docs/adr-business-finder-approach.md) and [business finder email signup](docs/business-finder-email-signup.md).
```

Comment:
So following on from a previous comment, this line could be: Read more about how links-based facets are handled